<!DOCTYPE html>
<html >
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1">
      <title>T000 Central Nervous System</title>
    
          <link rel="stylesheet" href="../_static/pygments.css" type="text/css" />
          <link rel="stylesheet" href="../_static/theme.css " type="text/css" />
      
      <!-- sphinx script_files -->
        <script data-url_root="../" id="documentation_options" src="../_static/documentation_options.js"></script>
        <script src="../_static/jquery.js"></script>
        <script src="../_static/underscore.js"></script>
        <script src="../_static/doctools.js"></script>

      
      <!-- bundled in js (rollup iife) -->
      <!-- <script src="../_static/theme-vendors.js"></script> -->
      <script src="../_static/theme.js" defer></script>
    
  <link rel="index" title="Index" href="../genindex.html" />
  <link rel="search" title="Search" href="../search.html" />
  <link rel="next" title="T001 Neuroblastoma" href="T001_nebla.html" />
  <link rel="prev" title="N000 Healthy Normal Tissue" href="N000_normal.html" /> 
  </head>

  <body>
    <div id="app">
    <div class="theme-container" :class="pageClasses"><navbar @toggle-sidebar="toggleSidebar">
  <router-link to="../index.html" class="home-link">
    
      <span class="site-name">Transcriptional Cancer Atlas Annotation</span>
    
  </router-link>

  <div class="links">
    <navlinks class="can-hide">

  
    <div class="nav-item">
      <a href="../index.html#welcome-to-the-annotation-of-the-transcriptional-cancer-atlas"
         class="nav-link  router-link-active">
         Annotation
      </a>
    </div>
  
    <div class="nav-item">
      <a href="../index.html#welcome-to-the-annotation-of-the-transcriptional-cancer-atlas"
         class="nav-link ">
         Dashboard
      </a>
    </div>
  
    <div class="nav-item">
      <a href="../index.html#welcome-to-the-annotation-of-the-transcriptional-cancer-atlas"
         class="nav-link ">
         F.A.Q.
      </a>
    </div>
  
    <div class="nav-item">
      <a href="../index.html#welcome-to-the-annotation-of-the-transcriptional-cancer-atlas"
         class="nav-link ">
         Contact us
      </a>
    </div>
  



    </navlinks>
  </div>
</navbar>

      
      <div class="sidebar-mask" @click="toggleSidebar(false)">
      </div>
        <sidebar @toggle-sidebar="toggleSidebar">
          
          <navlinks>
            

  
    <div class="nav-item">
      <a href="../index.html#welcome-to-the-annotation-of-the-transcriptional-cancer-atlas"
         class="nav-link  router-link-active">
         Annotation
      </a>
    </div>
  
    <div class="nav-item">
      <a href="../index.html#welcome-to-the-annotation-of-the-transcriptional-cancer-atlas"
         class="nav-link ">
         Dashboard
      </a>
    </div>
  
    <div class="nav-item">
      <a href="../index.html#welcome-to-the-annotation-of-the-transcriptional-cancer-atlas"
         class="nav-link ">
         F.A.Q.
      </a>
    </div>
  
    <div class="nav-item">
      <a href="../index.html#welcome-to-the-annotation-of-the-transcriptional-cancer-atlas"
         class="nav-link ">
         Contact us
      </a>
    </div>
  



            
          </navlinks><div id="searchbox" class="searchbox" role="search">
  <div class="caption"><span class="caption-text">Quick search</span>
    <div class="searchformwrapper">
      <form class="search" action="../search.html" method="get">
        <input type="text" name="q" />
        <input type="submit" value="Search" />
        <input type="hidden" name="check_keywords" value="yes" />
        <input type="hidden" name="area" value="default" />
      </form>
    </div>
  </div>
</div><div class="sidebar-links" role="navigation" aria-label="main navigation">
  
    <div class="sidebar-group">
      <p class="caption">
        <span class="caption-text"><a href="../index.html#welcome-to-the-annotation-of-the-transcriptional-cancer-atlas">Annotation</a></span>
      </p>
      <ul class="current">
        
          <li class="toctree-l1 ">
            
              <a href="N000_normal.html" class="reference internal ">N000 Healthy Normal Tissue</a>
            

            
          </li>

        
          <li class="toctree-l1 current">
            
              <a href="#" class="reference internal current">T000 Central Nervous System</a>
            

            
              <ul>
                
                  <li class="toctree-l2"><a href="#medulloblastoma" class="reference internal">Medulloblastoma</a></li>
                
                  <li class="toctree-l2"><a href="#separation-by-idh1-status" class="reference internal">Separation by IDH1 status</a></li>
                
                  <li class="toctree-l2"><a href="#bcor-altered-samples-and-ependymoma" class="reference internal">BCOR-altered Samples and Ependymoma</a></li>
                
                  <li class="toctree-l2"><a href="#idh-wild-type-glioma" class="reference internal">IDH wild-type glioma</a></li>
                
                  <li class="toctree-l2"><a href="#idh-mutant-glioma" class="reference internal">IDH-mutant glioma</a></li>
                
                  <li class="toctree-l2"><a href="#bibliography" class="reference internal">Bibliography</a></li>
                
              </ul>
            
          </li>

        
          <li class="toctree-l1 ">
            
              <a href="T001_nebla.html" class="reference internal ">T001 Neuroblastoma</a>
            

            
          </li>

        
          <li class="toctree-l1 ">
            
              <a href="T002_T003_mesoderm.html" class="reference internal ">T002 and T003 Mesodermal Tumour</a>
            

            
          </li>

        
          <li class="toctree-l1 ">
            
              <a href="T005_leuk.html" class="reference internal ">T005 Leukemia</a>
            

            
          </li>

        
      </ul>
    </div>
  
    <div class="sidebar-group">
      <p class="caption">
        <span class="caption-text"><a href="../index.html#welcome-to-the-annotation-of-the-transcriptional-cancer-atlas">Dashboard</a></span>
      </p>
      <ul class="">
        
          <li class="toctree-l1 ">
            
              <a href="../dash.html" class="reference internal ">Otter Dashboard</a>
            

            
          </li>

        
      </ul>
    </div>
  
    <div class="sidebar-group">
      <p class="caption">
        <span class="caption-text"><a href="../index.html#welcome-to-the-annotation-of-the-transcriptional-cancer-atlas">F.A.Q.</a></span>
      </p>
      <ul class="">
        
          <li class="toctree-l1 ">
            
              <a href="../faq.html" class="reference internal ">Frequently Asked Questions</a>
            

            
          </li>

        
      </ul>
    </div>
  
    <div class="sidebar-group">
      <p class="caption">
        <span class="caption-text"><a href="../index.html#welcome-to-the-annotation-of-the-transcriptional-cancer-atlas">Contact us</a></span>
      </p>
      <ul class="">
        
          <li class="toctree-l1 ">
            
              <a href="../contacts.html" class="reference internal ">Contact us</a>
            

            
          </li>

        
      </ul>
    </div>
  
</div>
        </sidebar>

      <page>
          <div class="body-header" role="navigation" aria-label="navigation">
  
  <ul class="breadcrumbs">
    <li><a href="../index.html">Docs</a> &raquo;</li>
    
    <li>T000 Central Nervous System</li>
  </ul>
  

  <ul class="page-nav">
  <li class="prev">
    <a href="N000_normal.html"
       title="previous chapter">← N000 Healthy Normal Tissue</a>
  </li>
  <li class="next">
    <a href="T001_nebla.html"
       title="next chapter">T001 Neuroblastoma →</a>
  </li>
</ul>
  
</div>
<hr>
          <div class="content" role="main" v-pre>
            
  <div class="section" id="t000-central-nervous-system">
<h1>T000 Central Nervous System<a class="headerlink" href="#t000-central-nervous-system" title="Permalink to this headline">¶</a></h1>
<p>Version: 0.1.0
<br/>
Last change: Feb 07, 2022</p>
<p>The <abbr title="Central Nervous System">CNS</abbr> tumors in our dataset exhibit three major divides:
glioma vs. non-gliomas, gliomas with wildtype or mutated IDH1,
and, in this last group, samples with and without hemizygous
codeletion of chromosome arms 1p and 19q (Fig. <a class="reference internal" href="#fig-cns1"><span class="std std-ref">CNS1</span></a>).
Past these major
divides, differences in histotypes and tumor phenotype, as well
as the transcriptional signals generated from specific mutations,
become increasingly important to differentiate sibling classes.</p>
<div class="figure align-default" id="fig-cns1">
<a class="reference internal image-reference" href="../_images/cns1.png"><img alt="Fig. CNS1" src="../_images/cns1.png" style="width: 600px;" /></a>
<p class="caption"><span class="caption-text">CNS1: On the left, a 2-dimensional UMAP projection of CNS tumors by gene expression, where several subtypes
found in the first layers of the hierarchy are highlighted with different colors. On the right, a list of all CNS subtypes identified
and their hierarchical relationship.</span><a class="headerlink" href="#fig-cns1" title="Permalink to this image">¶</a></p>
</div>
<div class="section" id="medulloblastoma">
<h2>Medulloblastoma<a class="headerlink" href="#medulloblastoma" title="Permalink to this headline">¶</a></h2>
<p>At the first level, we see the separation of medulloblastomas,
in <abbr title="Medulloblastoma">T027 MLBLA</abbr> (n = 29), from the rest of <abbr title="Central nervous system">CNS</abbr>
tumors <abbr title="Central nervous system, mix A">T026 CNS A</abbr> (n = 894)
(Fig. <a class="reference internal" href="#fig-cns1"><span class="std std-ref">CNS1</span></a>). Letters in the naming will be used in this setting to distinguish mixed
classes that maintain the same composition of their parent class, with the removal of
specific subtypes singled out into their sibling classes, as in this case. Interestingly,
we also note the presence of a single pineal parenchymal tumor in <abbr title="Central nervous system, mix A">T026</abbr>. Multi-scale clustering
allows us to observe the emergence of known subtypes from literature at deeper level of
this branch <a class="reference internal" href="#northcott2012" id="id1"><span>[Northcott2012]</span></a>. This class further splits into <abbr title="Medulloblastoma, G3 or G4">T058 MLBLA G3/G4</abbr> (n = 37)
(Fig. <a class="reference internal" href="#fig-cns1"><span class="std std-ref">CNS1</span></a>), a cluster of mixed G3 and G4 subtypes, with overexpression of <em>OTX2</em> (<abbr title="edgeR quasi-likelihood negative binomial generalized log-linear model">glmQLFTest</abbr> <abbr title="log-Fold Change">logFC</abbr> = 3.48, <abbr title="False Discovery Rate">FDR</abbr> = 6.368e-06)
and <em>FOXG1</em> (<abbr title="log-Fold Change">logFC</abbr> = 8.44, <abbr title="False Discovery Rate">FDR</abbr> = 4.026e-06), while <abbr title="Medulloblastoma, WNT or SHH">T059 MLBLA WNT/SSH</abbr> (n = 5), shows overexpression
of both <em>WNT</em> (<abbr title="single-sample GSEA from GSVA">ssGSEA</abbr> <a class="reference internal" href="#hanzelmann2013" id="id2"><span>[Hänzelmann2013]</span></a>, <abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.15,
<abbr title="Mann Whitney U test Benjamin-Hochberg adjusted p-value">MWU adj. p-val</abbr> = 6.74e-05) and <em>SHH</em> (<abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.42, <abbr title="Mann Whitney U test Benjamin-Hochberg adjusted p-value">MWU adj. p-val</abbr> = 2.02e-04)
pathways <a class="reference internal" href="#kanehisa2000" id="id3"><span>[Kanehisa2000]</span></a>. While samples of the G3 and G4 subtypes are then separated
at the next level into <abbr title="Medulloblastoma, G4">T060 MLBLA G4</abbr> (n = 9) and <abbr title="Medulloblastoma, G3">T061 MLBLA G3</abbr>
(n = 15), the population of <abbr title="Medulloblastoma, WNT or SHH">T059</abbr> is
well below our set cut-off, preventing RACCOON from dividing <em>WNT</em> and <em>SHH</em> subtypes. <abbr title="Medulloblastoma, G4">T060</abbr> overexpresses
genes of the G4 subtype, including <em>SNCAIP</em> (<abbr title="log-Fold Change">logFC</abbr> = 5.68, <abbr title="False Discovery Rate">FDR</abbr> = 1.11e-05), <em>DIRAS3</em> (<abbr title="log-Fold Change">logFC</abbr> = 4.35,
<abbr title="False Discovery Rate">FDR</abbr> = 2.351e-06), <em>KCNA1</em> (<abbr title="log-Fold Change">logFC</abbr> = 4.19, <abbr title="False Discovery Rate">FDR</abbr> = 3.684e-04), and <em>RND1</em> (<abbr title="log-Fold Change">logFC</abbr> = 3.26, <abbr title="False Discovery Rate">FDR</abbr> = 1.542e-04),
while <abbr title="Medulloblastoma, G3">T061</abbr> overexpresses genes upregulated in the G3 subtype,
such as <em>PDE6H</em> (<abbr title="log-Fold Change">logFC</abbr> = -6, <abbr title="False Discovery Rate">FDR</abbr> = 6.038e-04), <em>GNGT1</em> (<abbr title="log-Fold Change">logFC</abbr> = -6.1, <abbr title="False Discovery Rate">FDR</abbr> = 2.651e-04),
and <em>NPR3</em> (<abbr title="log-Fold Change">logFC</abbr> = -5.71, <abbr title="False Discovery Rate">FDR</abbr> = 4.824e-04).</p>
</div>
<div class="section" id="separation-by-idh1-status">
<h2>Separation by IDH1 status<a class="headerlink" href="#separation-by-idh1-status" title="Permalink to this headline">¶</a></h2>
<p>Following the remainder of CNS tumors after the removal of medulloblastomas,
we observe the separation of gliomas without IDH1 mutations, which form <abbr title="Central nervous system tumors, IDH wild type">T028 CNS IDHwt</abbr> (n = 406)
from samples with IDH1 mutations (19/222 vs 417/433, <abbr title="χ2 test p-value">χ2 p-val</abbr> &lt; 2.2e-16), which form <abbr title="Central nervous system tumors, IDH-mutant">T029 CNS IDHmut</abbr> (n = 488) (Fig. <a class="reference internal" href="#fig-cns1"><span class="std std-ref">CNS1</span></a>).
The latter has patients with lower median age (49.00 vs 38.00 y.o., <abbr title="Mann Whitney U test p-value">MWU p-val</abbr> = 2.04e-3), but <abbr>T028</abbr> has a considerably higher proportion
of pediatric patients (40.06% vs. 27.05%, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 2.40e-05). Furthermore, <abbr title="Central nervous system tumors, IDH wild type">T028</abbr> displays patients with significantly worse survival
(<abbr>lrt p-val (Kaplan-Meier log rank test p-value)`</abbr>  = 1.57e-50 at 6423 days) in line with literature <a class="reference internal" href="#hartmann2010" id="id4"><span>[Hartmann2010]</span></a> reaching median overall
survival (OS) at only 448 days compared to <abbr title="Central nervous system tumors, IDH-mutant">T029</abbr> at 2907 (Fig. <a class="reference internal" href="#fig-cns2"><span class="std std-ref">CNS2</span></a>) <a class="reference internal" href="#park2016" id="id5"><span>[Park2016]</span></a>, <a class="reference internal" href="#steponaitis2016" id="id6"><span>[Steponaitis2016]</span></a>, <a class="reference internal" href="#cimino2018" id="id7"><span>[Cimino2018]</span></a>, <a class="reference internal" href="#hernandez2010" id="id8"><span>[Hernández2010]</span></a>.</p>
<div class="figure align-default" id="fig-cns2">
<a class="reference internal image-reference" href="../_images/cns2.png"><img alt="Fig. CNS2" src="../_images/cns2.png" style="width: 300px;" /></a>
<p class="caption"><span class="caption-text">CNS2: Overall survival time curves of wild-type and mutant IDH tumors.</span><a class="headerlink" href="#fig-cns2" title="Permalink to this image">¶</a></p>
</div>
</div>
<div class="section" id="bcor-altered-samples-and-ependymoma">
<h2>BCOR-altered Samples and Ependymoma<a class="headerlink" href="#bcor-altered-samples-and-ependymoma" title="Permalink to this headline">¶</a></h2>
<p>Along the IDH1 wild-type branch <abbr title="Central nervous system tumors, IDH wild type">T028</abbr> we then observe the separation of gliomas and
glioblastomas in <abbr title="Glioma, IDH wild-type">T030 GLI IDHwt</abbr> from ependymomas in
<abbr title="Ependymoma">T032 EPDY</abbr> and samples with lesions of the BCL-6 corepressor protein gene, <em>BCOR</em>, <abbr title="Central nervous system with BCOR alterations, primitive neuroectodermal tumors">T031 CNS BCOR/PNET</abbr>
(Fig. <a class="reference internal" href="#fig-cns1"><span class="std std-ref">CNS1</span></a>, Fig. <a class="reference internal" href="#fig-cns3"><span class="std std-ref">CNS3a</span></a>).
<abbr title="Glioma, IDH wild-type">T030 GLI IDHwt</abbr> (n = 364) includes the vast majority of gliomas and glioblastomas without mutations of IDH1 and is the oldest class
(median age = 52 y.o. <abbr title="Kruskal–Wallis one-way analysis of variance test p-value">KW p-val</abbr> = 5.72e-10).
<abbr title="Central nervous system with BCOR alterations, primitive neuroectodermal tumors">T031 CNS BCOR/PNET</abbr> is a peculiarly small cluster (n = 12)
comprised of heterogeneous diagnoses. It includes a variety of brain and
CNS tumors, including ependymomas, primitive neuroectodermal tumors (PNET), gliomas, an embryonal tumor with multi-layered rosettes,
and a handful of solid tumors - several possibly misdiagnosed as Ewing sarcoma - and one infantile fibrosarcoma (Fig. 5d). All samples
are from pediatric patients, with a median age of 4.5 y.o. This cluster is characterized by an overexpression of <em>BCOR</em> (<abbr title="median log-fold change)= 4.38, :abbr:`FDR (False Discovery Rate">medLogFC</abbr> ≤ 2.94e-41) (Fig. <a class="reference internal" href="#fig-cns3"><span class="std std-ref">CNS3c</span></a>).</p>
<p>BCOR participates in a range of chromatin altering activities including binding to histone acetylases and chromatin-altering complexes,
namely polycomb group complexe <a class="reference internal" href="#gearhart2006" id="id9"><span>[Gearhart2006]</span></a>. Alterations of these genes, many of which consist of fusions or internal tandem duplications
(ITD) (Fig. <a class="reference internal" href="#fig-cns3"><span class="std std-ref">CNS3b</span></a>), have been well characterized in both soft tissue tumors and a recently defined group of CNS neoplasms:  high-grade
neuroepithelial tumors of the central nervous system (CNS HGNET-BCOR) <a class="reference internal" href="#sturm2016" id="id10"><span>[Sturm2016]</span></a>. Gene set enrichment analyses revealed significant
upregulation of both WNT (<abbr>medNES median Normalized Enrichment Score)</abbr> ≥ 1.35, <abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 3.83e-09,
<abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 1.00e-04) <a class="reference internal" href="T002_T003_mesoderm.html#liberzon2015" id="id11"><span>[Liberzon2015]</span></a> and SHH
(<abbr>medNES median Normalized Enrichment Score)</abbr> ≥ 1.51, <abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 6.16e-09, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 1.00e-04) pathways <a class="reference internal" href="T001_nebla.html#nishimura2001" id="id12"><span>[Nishimura2001]</span></a>, as well as basal cell carcinoma pathways
(<abbr>medNES median Normalized Enrichment Score)</abbr> ≥ 1.70, <abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 2.84e-20, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 1.00e-04) <a class="reference internal" href="#kanehisa2000" id="id13"><span>[Kanehisa2000]</span></a> in line with what is reported in literature.
We also observe significant overexpression of <em>NTRK3</em> (<abbr title="median log-fold change">medLogFC</abbr> = 2.45, <abbr title="False Discovery Rate">FDR</abbr> ≤ 1.7e-16), but not <em>NTRK2</em> (<abbr title="False Discovery Rate">FDR</abbr> ≤ 5.847e-01) and <em>NTRK1</em> (<abbr title="False Discovery Rate">FDR</abbr> ≤ 9.063e-01)
in <abbr title="Central nervous system with BCOR alterations, primitive neuroectodermal tumors">T031</abbr> vs. <abbr title="Glioma, IDH wild-type">T030</abbr> and <abbr title="Ependymoma">T032</abbr>, as commonly described in BCOR-ITD sarcomas <a class="reference internal" href="#kao2018" id="id14"><span>[Kao2018]</span></a>, <a class="reference internal" href="#kao2020" id="id15"><span>[Kao2020]</span></a>.
Finally, <abbr title="Ependymoma">T032 EPDY</abbr> (n = 30) is comprised almost exclusively of ependymomas. It is the cluster with the youngest patients, with a median age
of 2.64 y.o. No subtypes are identified, possibly due to the limits in the reference dataset population.</p>
<div class="figure align-default" id="fig-cns3">
<a class="reference internal image-reference" href="../_images/cns3.png"><img alt="Fig. CNS3" src="../_images/cns3.png" style="width: 600px;" /></a>
<p class="caption"><span class="caption-text">CNS3: Summary of the findings relating to BCOR-mutated and CIC-mutated tumors.
A) 2-dimensional UMAP projection of CNS tumors by gene expression, where a few representative classes are shown with shades of blue and green.
The BCOR-mutated class is highlighted in orange (T031). B) Diagram representing the archetypical BCOR-ITD and BCOR-CCNB3 rearrangements.
C) BCOR expression distribution across representative CNS classes, showing a clear overexpression in BCOR-mutated samples (T031).
D) The idiosyncratic transcriptional profile of BCOR mutations is sufficient to overcome the cell-of-origin attraction during the clustering process.
The ratio of tumor types within T031, shows that while it is mostly composed of CNS tumors, sarcomas are also found in this class.</span><a class="headerlink" href="#fig-cns3" title="Permalink to this image">¶</a></p>
</div>
</div>
<div class="section" id="idh-wild-type-glioma">
<h2>IDH wild-type glioma<a class="headerlink" href="#idh-wild-type-glioma" title="Permalink to this headline">¶</a></h2>
<p>At the next level, we observe the separation between a small pediatric cluster <abbr title="Glioma, low-grade, pediatric">T033 GLI LG PED</abbr> (n = 63)
and a much larger adult class <abbr title="Glioma high-grade">T034 GLI HG</abbr> (n=301) (Fig. <a class="reference internal" href="#fig-cns1"><span class="std std-ref">CNS1</span></a>). Both contain mixed diagnoses but with a
strong majority of samples labelled as gliomas. There’s a significant difference in age, with <abbr title="Glioma, low-grade, pediatric">T033</abbr> having a
population with a median age of only 9.00 y.o. versus <abbr title="Glioma high-grade">T034</abbr> with 56.00 y.o. (<abbr title="Mann-Whitney U test p-value">MWU p-val</abbr> = 4.00e-20).
<abbr title="Glioma high-grade">T034</abbr> is characterized
by significant upregulation of <em>HOX</em> genes (36/39 <abbr title="False Discovery Rate">FDR</abbr> &lt; 0.05), particularly <em>HOXD9</em> (<abbr title="log-Fold Change">logFC</abbr> = -5.03, <abbr title="False Discovery Rate">FDR</abbr> = 1.20e-23) and HOXA5
(<abbr title="log-Fold Change">logFC</abbr> = -6.18, <abbr title="False Discovery Rate">FDR</abbr> = 2.40e-29)  <a class="reference internal" href="#tabuse2011" id="id16"><span>[Tabuse2011]</span></a>, <a class="reference internal" href="#cimino2018" id="id17"><span>[Cimino2018]</span></a> which have been associated with cancer cell survival
and proliferation in gliomas. Together with overexpression of <em>VEGFA</em> <a class="reference internal" href="#xu2017" id="id18"><span>[Xu2017]</span></a> (<abbr title="log-Fold Change">logFC</abbr> = -1.04, <abbr title="False Discovery Rate">FDR</abbr> = 2.15e-05),
a marker of poor survival, and glioma stemness genes <em>TERT</em> and <em>EGFR</em> (<abbr title="False Discovery Rate">FDR</abbr> ≤ 1.00e-28) <a class="reference internal" href="#beck2011" id="id19"><span>[Beck2011]</span></a>, this profile suggests <abbr title="Glioma high-grade">T034</abbr> to be a class of
high-grade gliomas and glioblastoma multiforme, while <abbr title="Glioma, low-grade, pediatric">T033</abbr> to be a largely pediatric, low-grade glioma class, though all samples
from the TCGA are astrocytomas (6/6 vs 50/232, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 6.74e-05). This is supported by <abbr title="Glioma, low-grade, pediatric">T033</abbr> being enriched for grade II (3/6 vs 10/232,
<abbr title="χ2 test p-value">χ2 p-val</abbr> = 7.73e-05) samples, with <abbr title="Glioma high-grade">T034</abbr> being enriched for grade IV samples (0/6 vs 155/232, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 3.11e-03).  However, we are unable
to confirm differences in survival due to a lack of clinical annotation of samples in <abbr title="Glioma, low-grade, pediatric">T033</abbr>.</p>
<p>The glioma subtypes run much deeper along complex hierarchical paths. At the next level, <abbr title="Glioma high-grade">T034</abbr> splits into <abbr title="Glioma, high-grade, Chr7 gain and Chr10 loss">T035 GLI HG LOH c7/10</abbr>  (n =236)
and <abbr title="Glioma, high-grade, proneural subtype">T036 GLI HG PRON</abbr>  (N = 65) (Fig. <a class="reference internal" href="#fig-cns4"><span class="std std-ref">CNS4</span></a>). Both are mixed glioma and glioblastoma groups. We also observe a significant difference in age
(median 58.00 vs 35.00 y.o. <abbr title="Mann-Whitney U test p-value">MWU p-val</abbr> = 8.76e-06) and pediatric composition (13.56% vs 50.77%, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 3.27e-10).
There is no difference in overall survival between the groups (<abbr title="Kaplan-Meier log rank test p-value">lrt p-val</abbr> = 8.23e-02 at 6423 days) <a class="reference internal" href="#ceccarelli2016" id="id20"><span>[Ceccarelli2016]</span></a>.
<abbr title="Glioma, high-grade, Chr7 gain and Chr10 loss">T035</abbr> contains almost all samples of the classical (85/185 vs. 1/29, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 3.527e-05) and mesenchymal (87/185 vs. 3/29, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 4.343e-04)
expression subtypes, while <abbr title="Glioma, high-grade, proneural subtype">T036</abbr> is almost wholly composed of the proneural subtypes (2/185 vs. 24/29, <abbr title="χ2 test p-value">χ2 p-val</abbr> &lt; 2.2e-16);
although the majority of neural type samples are also found in <abbr title="Glioma, high-grade, Chr7 gain and Chr10 loss">T035</abbr>, the difference is not significant (11/185 vs. 1/29,
<abbr title="χ2 test p-value">χ2 p-val</abbr> = 0.9128) <a class="reference internal" href="#ceccarelli2016" id="id21"><span>[Ceccarelli2016]</span></a>, <a class="reference internal" href="#brennan2013" id="id22"><span>[Brennan2013]</span></a>. <abbr title="Glioma, high-grade, Chr7 gain and Chr10 loss">T035</abbr> shows significant overexpression of <em>SAA1</em> (<abbr title="log-Fold Change">logFC</abbr> = 4.84,
<abbr title="False Discovery Rate">FDR</abbr> = 2.869e-16), <em>MEOX2</em> (<abbr title="log-Fold Change">logFC</abbr> = 4.79, <abbr title="False Discovery Rate">FDR</abbr> = 8.46e-22), <em>CHI3L1</em> (<abbr title="log-Fold Change">logFC</abbr> = 3.5,
<abbr title="False Discovery Rate">FDR</abbr> = 6.93e-20), <em>S100A4</em> (<abbr title="log-Fold Change">logFC</abbr> = 2.04, <abbr title="False Discovery Rate">FDR</abbr> = 1.26e-18)
and <em>ANXA1</em> (<abbr title="log-Fold Change">logFC</abbr> = 2.68, <abbr title="False Discovery Rate">FDR</abbr> = 1.18e-37), all associated with poor survival <a class="reference internal" href="#tachon2019" id="id23"><span>[Tachon2019]</span></a>, <a class="reference internal" href="#xu2017" id="id24"><span>[Xu2017]</span></a> ,and has a considerably
higher leukocyte content than <abbr title="Glioma, high-grade, proneural subtype">T036</abbr> (0.190 vs. 0.059, <abbr title="Mann-Whitney U test p-value">MWU p-val</abbr> = 1.42e-08) <a class="reference internal" href="T002_T003_mesoderm.html#thorsson2018" id="id25"><span>[Thorsson2018]</span></a>. In turn, <abbr title="Glioma, high-grade, proneural subtype">T036</abbr> samples overexpress
<em>PDGFRA</em> (<abbr title="log-Fold Change">logFC</abbr> = -2.8, <abbr title="False Discovery Rate">FDR</abbr> = 3.80e-34), a marker of the proneuronal expression type(Brennan2013). <abbr title="Glioma, high-grade, Chr7 gain and Chr10 loss">T035</abbr> contains more <em>TP53</em>
mutants (<abbr title="χ2 test p-value">χ2 p-val</abbr> =2.11-02), and is also enriched for genesets concerning loss of heterozygosity (LOH) of regions implicated in
gliomagenesis (<abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.32, <abbr title="Mann Whitney U test Benjamin-Hochberg adjusted p-value">MWU adj. p-val</abbr> = 2.15e-06) <a class="reference internal" href="#roversi2006" id="id26"><span>[Roversi2006]</span></a>, suggesting it contains samples with gain of
chromosome 7 and loss of chromosome 10. This is further supported by its overexpression of <em>EGFR</em> (<abbr title="log-Fold Change">logFC</abbr> = 3.47, <abbr title="False Discovery Rate">FDR</abbr> = 1.18e-18)
and is in line with literature, in which classical <abbr title="Glioblastoma">GBM</abbr> samples tend to harbour these lesions. Indeed, <abbr title="Glioma, high-grade, Chr7 gain and Chr10 loss">T035</abbr> is highly enriched for
tumors with gain chr7/loss chr10, confirmed by clinical data (139/200 vs. 15/35, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 4.146e-03) <a class="reference internal" href="#ceccarelli2016" id="id27"><span>[Ceccarelli2016]</span></a>.
<abbr title="Glioma, high-grade, proneural subtype">T036</abbr> contains a greater proportion of <em>ATRX</em>-mutant tumors (9/194 vs. 10/24, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 7.31e-06) <a class="reference internal" href="#ceccarelli2016" id="id28"><span>[Ceccarelli2016]</span></a>.</p>
<div class="figure align-default" id="fig-cns4">
<a class="reference internal image-reference" href="../_images/cns4.png"><img alt="Fig. CNS4" src="../_images/cns4.png" style="width: 400px;" /></a>
<p class="caption"><span class="caption-text">CNS4: 2-dimensional UMAP projection of subtypes of IDH-mutant gliomas.
Samples with reported gain of chromosome 7 and loss of chromosome 10 are shown as empty circles.</span><a class="headerlink" href="#fig-cns4" title="Permalink to this image">¶</a></p>
</div>
<p>Glioblastomas and high-grade gliomas separate at the next level within <abbr title="Glioma, high-grade, proneural subtype">T036</abbr> (Fig. <a class="reference internal" href="#fig-cns1"><span class="std std-ref">CNS1</span></a>). We observe <abbr title="Glioma, high-grade, or glioblastoma, proneural subtype">T042 GLI HG/GBM PRON</abbr>  (n = 48)
carrying glioblastomas mostly of the proneuronal subtype and <abbr title="Glioma, high-grade, pediatric, H3.3 mutant">T043 GLI HG PED H3.3mut</abbr> (n =19) with the rest of the samples, primarily
marked as high-grade gliomas from St. Jude’s (<abbr title="χ2 test p-value">χ2 p-val</abbr> = 8.75e-14) (Fig. <a class="reference internal" href="#fig-cns1"><span class="std std-ref">CNS1</span></a>). The two classes also differ significantly in age,
with <abbr title="Glioma, high-grade, or glioblastoma, proneural subtype">T042</abbr> having patients with a median age of 44.5 y.o. while <abbr title="Glioma, high-grade, pediatric, H3.3 mutant">T043</abbr>
has a median age of 5.85 y.o. (<abbr title="Mann-Whitney U test p-value">MWU p-val</abbr> 3.88e-05). In fact, <abbr title="Glioma, high-grade, pediatric, H3.3 mutant">T043</abbr>
is the cluster with the youngest group of patients within the entire cohort of both gliomas and gliobastomas and is one of only two
clusters with &gt;90% pediatric composition, the other being <abbr title="Glioma, low-grade, pediatric">T033 GLI LG PED</abbr>. Given that it’s a majority pediatric cluster whose parent
cluster demonstrates very poor survival, <abbr title="Glioma, high-grade, pediatric, H3.3 mutant">T043</abbr> may represent H3.3 (<em>H3F3A</em>) mutated tumors. Support for this hypothesis comes from
enrichment of gene sets involving H3.3 mutation (here nominally K27M) between <abbr title="Glioma, high-grade, pediatric, H3.3 mutant">T043</abbr> and <abbr title="Glioma, high-grade, or glioblastoma, proneural subtype">T042</abbr>
(<abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.73, <abbr title="Mann Whitney U test Benjamin-Hochberg adjusted p-value">MWU adj. p-val</abbr> = 3.39e-02) (Fig. <a class="reference internal" href="#fig-cns5"><span class="std std-ref">CNS5</span></a>) <a class="reference internal" href="#larson2019" id="id29"><span>[Larson2019]</span></a>.
Notch signalling (<abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.05, <abbr title="Mann Whitney U test Benjamin-Hochberg adjusted p-value">MWU adj. p-val</abbr> = 2.31e-02) and neural differentiation
(<abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.04, <abbr title="Mann Whitney U test Benjamin-Hochberg adjusted p-value">MWU adj. p-val</abbr> =6.10e-03) <a class="reference internal" href="T002_T003_mesoderm.html#schaefer2009" id="id30"><span>[Schaefer2009]</span></a>.
genesets are also enriched in <abbr title="Glioma, high-grade, pediatric, H3.3 mutant">T043</abbr> and are a feature of these tumors. <a class="reference internal" href="#paugh2011" id="id31"><span>[Paugh2011]</span></a>, <a class="reference internal" href="#chen2020" id="id32"><span>[Chen2020]</span></a> Going back up along the hierarchy,
<abbr title="Glioma, high-grade, Chr7 gain and Chr10 loss">T035 GLI HG LOH c7/10</abbr>  also splits in two (Fig. <a class="reference internal" href="#fig-cns1"><span class="std std-ref">CNS1</span></a>), with <abbr title="Glioma, high-grade, neural differentiation">T037 GLI HG NEUR DIFFhigh</abbr>  (n = 93) being comprised of gliomas and glioblastomas of
the classical (49/68 vs. 36/117, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 1.29e-07) and neural subtypes (8/68 vs. 3/117, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 2.58e-02) and <abbr title="Glioma, high-grade, or glioblastoma, mesenchymal or classical subtype">T038 GLI HG/GBM MES/CLASS</abbr>  (n = 143)
carrying a mixture of glioblastomas multiforme subtypes. <abbr title="Glioma, high-grade, neural differentiation">T037</abbr> is composed of a majority of astrocytomas (34/78 vs. 10/118), <abbr title="χ2 test p-value">χ2 p-val</abbr> = 3.13e-08)
while <abbr title="Glioma, high-grade, or glioblastoma, mesenchymal or classical subtype">T038</abbr> contains a majority of glioblastomas (31/78 vs. 100/118, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 9.30e-11).
Interestingly, while almost all of the gliomas in <abbr title="Glioma, high-grade, or glioblastoma, mesenchymal or classical subtype">T038</abbr>
are marked as IDH1 wild type, a handful of samples are IDH1-mutant (0/84 vs. 10/114, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 1.40e-2), suggesting these may be passenger rather
than driver mutations. <abbr title="Glioma, high-grade, or glioblastoma, mesenchymal or classical subtype">T038</abbr> also has significantly higher leukocyte fraction (0.178 vs. 0.248, <abbr title="Mann-Whitney U test p-value">MWU p-val</abbr> = 1.77e-02). There are no differences in
proportion of gain chr7/loss chr10 samples (<abbr title="χ2 test p-value">χ2 p-val</abbr> = 3.80e-01).</p>
<div class="figure align-default" id="fig-cns5">
<a class="reference internal image-reference" href="../_images/cns5.png"><img alt="Fig. CNS5" src="../_images/cns5.png" style="width: 500px;" /></a>
<p class="caption"><span class="caption-text">CNS5: Per sample Normalized Enrichment Score (NES) distributions of gene sets characterizing the H3.3-mutant glioma subtype.</span><a class="headerlink" href="#fig-cns5" title="Permalink to this image">¶</a></p>
</div>
<p>The remaining subtypes are found in the child classes of <abbr title="Glioma, high-grade, or glioblastoma, mesenchymal or classical subtype">T038</abbr> (Fig. <a class="reference internal" href="#fig-cns1"><span class="std std-ref">CNS1</span></a>): we find the majority of classical samples (31/34 vs 2/27, vs 1/43,
<abbr title="χ2 test p-value">χ2 p-val</abbr> &lt; 2.2e-16) in <abbr title="Glioma, high-grade, or glioblastoma, classical subtype">T039 GLI HG/GBM CLASS</abbr>  (n = 37), mesenchymal subtype samples in both <abbr title="Glioma, high-grade, or glioblastoma, mesenchymal subtype">T040</abbr> GLI HG/GBM MES (n = 36) and
<abbr title="Glioma, high-grade, or glioblastoma, neural subtype">T041 GLI HG/GBM NEUR</abbr> (n=57) (3/34 vs. 24/27 vs. 38/43, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 3.28e-14). <abbr title="Glioma, high-grade, or glioblastoma, mesenchymal subtype">T040</abbr> contains two concurrent <em>PIK3CA</em> and
<em>NF1</em>-mutated samples (<abbr title="χ2 test p-value">χ2 p-val</abbr> = 2.73e-02), <em>NF1</em> mutations are typical of mesenchymal <abbr title="Glioblastoma">GBM</abbr> <a class="reference internal" href="#fadhlullah2019" id="id33"><span>[Fadhlullah2019]</span></a>. <abbr title="Glioma, high-grade, or glioblastoma, neural subtype">T041</abbr> inherits all
<em>IDH1</em>-mutants (0/34 vs. 0/26 vs. 10/42, <abbr title="χ2 test p-value">χ2 p-val</abbr>  = 3.64e-04) and is enriched for <em>TP53</em> mutants (0/5 vs. 0/5 vs 5/6, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 1.38e-2).
Patients at <abbr title="Glioma, high-grade, or glioblastoma, classical subtype">T039</abbr> have the best overall survival, reaching median <abbr title="Overall Survival">OS</abbr> at 375 days post diagnosis, while those in <abbr title="Glioma, high-grade, or glioblastoma, mesenchymal subtype">T040</abbr> have the worst,
reaching median <abbr title="Overall Survival">OS</abbr> at 225 days (<abbr title="Kaplan-Meier log rank test p-value">lrt p-val</abbr> = 3.44e-02 at 2549 days). These clusters differ in their share of <em>TERT</em> promoter mutations and
<em>ATRX</em> mutations when available, respectively, with <abbr title="Glioma, high-grade, or glioblastoma, classical subtype">T039</abbr> and <abbr title="Glioma, high-grade, or glioblastoma, mesenchymal subtype">T040</abbr> comprised of samples with <em>TERT</em> promoter mutants (8/8 vs. 9/9 vs. 4/9, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 2.89e-03)
while <abbr title="Glioma, high-grade, or glioblastoma, neural subtype">T041</abbr> contains all <em>ATRX</em>-mutants (0/33 vs. 0/25 vs. 7/40, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 4.23e-03). Examination of telomere maintenance pathways reveals samples
with relevant data in <abbr title="Glioma, high-grade, or glioblastoma, neural subtype">T041</abbr> to be driven more by <em>ATRX</em> mutations (0/8 vs. 0/8, 4/5, <abbr title="χ2 test p-value">χ2 p-val</abbr> =1.4513-02) while its siblings are wholly driven by
<em>TERT</em> mutations (8/8, 8/8, 4/5, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 3.87e-03). Analysis of gene sets for relevant pathways shows <abbr title="Glioma, high-grade, or glioblastoma, mesenchymal subtype">T040</abbr> to be enriched for mesenchymal
<abbr title="Glioblastoma">GBM</abbr> over its siblings (<abbr title="median Normalized Enrichment Score">medNES</abbr> ≥ 1.27, <abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 8.91e-14, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 1e-04),
while <abbr title="Glioma, high-grade, or glioblastoma, neural subtype">T041</abbr> is enriched for neural
<abbr title="Glioblastoma">GBM</abbr> (<abbr title="median Normalized Enrichment Score">medNES</abbr> ≥ 1.38, <abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 5.59e-14, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 1e-04),
suggesting this subtype has a more neural than mesenchymal identity.
This is further supported by <abbr title="Glioma, high-grade, or glioblastoma, neural subtype">T041</abbr>’s inheritance of the majority of neural (0/34 vs. 1/27 vs. 2/43, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 4.60e-01) and proneural samples
(0/34 vs. 0/27 vs. 2/43, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 2.35e-01), though neither reach significance.</p>
<p>While it is surprising to see two unrelated clusters of glioblastomas containing large populations of classical expression subtype glioblastomas,
<abbr title="Glioma, high-grade, neural differentiation">T037</abbr> and <abbr title="Glioma, high-grade, or glioblastoma, classical subtype">T039</abbr>, closer examination reveals <abbr title="Glioma, high-grade, or glioblastoma, classical subtype">T039</abbr> to represent a bona fide classical <abbr title="Glioblastoma">GBM</abbr> subtype, supported by significant enrichment of classical
glioblastoma gene signatures (<abbr title="median Normalized Enrichment Score">medNES</abbr>  1.15, <abbr title="Mann Whitney U test Benjamin-Hochberg adjusted p-value">MWU adj. p-val</abbr> = 1.61e-08), and by its higher proportion of classical samples
(31/34 vs. 49/68, <abbr title="Fisher Exact Test p-value">FET p-val</abbr> = 3.94e-02) (Fig. <a class="reference internal" href="#fig-cns6"><span class="std std-ref">CNS6</span></a>). <abbr title="Glioma, high-grade, neural differentiation">T037</abbr> is instead enriched for neural (<abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.79, <abbr title="Mann Whitney U test Benjamin-Hochberg adjusted p-value">MWU adj. p-val</abbr> = 1.87e-13)
and proneural signatures (<abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.15, <abbr title="Mann Whitney U test Benjamin-Hochberg adjusted p-value">MWU adj. p-val</abbr> = 8.81e-05) over <abbr title="Glioma, high-grade, or glioblastoma, classical subtype">T039</abbr> (Fig. <a class="reference internal" href="#fig-cns6"><span class="std std-ref">CNS6</span></a>). Furthermore, the presence of a sizeable
astrocytoma/glioma component in <abbr title="Glioma, high-grade, neural differentiation">T037</abbr> suggests it may represent a more “mixed” phenotype of glioma than its sibling <abbr title="Glioma, high-grade, or glioblastoma, mesenchymal or classical subtype">T038</abbr>, whose
children separate into histotype-specific component clusters of <abbr title="Glioblastoma">GBM</abbr>. Indeed, <abbr title="Glioma, high-grade, neural differentiation">T037</abbr> is enriched for neural and proneural signatures against
all children of <abbr title="Glioma, high-grade, or glioblastoma, mesenchymal or classical subtype">T038</abbr> (<abbr title="median Normalized Enrichment Score">medNES</abbr> ≤ 1.04, <abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> ≤ 4.07e-16) (Fig. <a class="reference internal" href="#fig-cns6"><span class="std std-ref">CNS6</span></a>).
We hypothesize <abbr title="Glioma, high-grade, neural differentiation">T037</abbr> represents a more neurally differentiated
class, transcending canonical subtyping. This is further supported by enrichment of genes pertaining to neural development (<abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.06,
<abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 7.01e-14, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 0.05) and differentiation (<abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.11,
<abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 2.24e-12, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 0.001) <a class="reference internal" href="T002_T003_mesoderm.html#tgoc2019" id="id34"><span>[TGOC2019]</span></a>, <a class="reference internal" href="T005_leuk.html#ashburner2000" id="id35"><span>[Ashburner2000]</span></a>
in <abbr title="Glioma, high-grade, neural differentiation">T037</abbr> with respect to <abbr title="Glioma, high-grade, or glioblastoma, classical subtype">T039</abbr>, <abbr title="Glioma, high-grade, or glioblastoma, mesenchymal subtype">T040</abbr>, and <abbr title="Glioma, high-grade, or glioblastoma, neural subtype">T041</abbr> (Fig. <a class="reference internal" href="#fig-cns6"><span class="std std-ref">CNS6</span></a>).</p>
<div class="figure align-default" id="fig-cns6">
<a class="reference internal image-reference" href="../_images/cns6.png"><img alt="Fig. CNS6" src="../_images/cns6.png" style="width: 800px;" /></a>
<p class="caption"><span class="caption-text">CNS6: Per sample Normalized Enrichment Score (NES) distributions of gene sets characterizing gliomas with high neural differentiation.</span><a class="headerlink" href="#fig-cns6" title="Permalink to this image">¶</a></p>
</div>
</div>
<div class="section" id="idh-mutant-glioma">
<h2>IDH-mutant glioma<a class="headerlink" href="#idh-mutant-glioma" title="Permalink to this headline">¶</a></h2>
<p>Along the alternative branch hosting IDH-mutant gliomas (<abbr title="Central nervous system tumors, IDH-mutant">T029</abbr>) we find that the hemizygous codeletion of chromosome arms
1p and 19q is a major driver in the separation of classes: we find samples with codeletion in <abbr>T044 GLI IHDmut CODEL Glioblastoma, IDH-mutant 1p19q codeletion)</abbr> (n = 270)
and samples without codeletion in <abbr title="Glioblastoma, IDH-mutant without 1p19q codeletion">T045 GLI IDHmut noCODEL</abbr> (n = 218) (168/ 221 vs. 1/213, <abbr title="χ2 test p-value">χ2 p-val</abbr> &lt; 2.2e-16) (Fig. <a class="reference internal" href="#fig-cns1"><span class="std std-ref">CNS1</span></a>, c).
Though <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion">T044</abbr> has significantly older patients (median age 40 vs. 36 y.o. <abbr title="Mann-Whitney U test p-value">MWU p-val</abbr> = 2.56e-03), it has a slightly larger pediatric
population (28.14% vs. 25.69%). According to the clinical information from TGCA, <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion">T044</abbr> contains significantly more tumors with <em>TERT</em>
promoter mutations (89/125 vs. 6/124, <abbr title="χ2 test p-value">χ2 p-val</abbr> &lt; 2.2e-16), while <abbr title="Glioblastoma, IDH-mutant without 1p19q codeletion">T045</abbr> is enriched for <em>ATRX</em> mutants (23/ 221 vs. 155/211, <abbr title="χ2 test p-value">χ2 p-val</abbr> &lt; 2.2e-16).
<abbr title="Glioblastoma, IDH-mutant 1p19q codeletion">T044</abbr> contains a majority of oligodendrogliomas (134/198 vs. 28/183, <abbr title="χ2 test p-value">χ2 p-val</abbr> &lt; 2.2e-16) while <abbr title="Glioblastoma, IDH-mutant without 1p19q codeletion">T045</abbr> contains a majority of astrocytomas
(17/198 vs. 97/183, <abbr title="χ2 test p-value">χ2 p-val</abbr> &lt; 2.2e-16). However, despite differences in codeletion status, we find no difference in overall survival
between the two groups (<abbr title="Kaplan-Meier log rank test p-value">lrt p-val</abbr> = 3.78e-01 at 5546 days).
Following along <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion">T044</abbr>, we observe the singling out of a small set of low-grade gliomas (n = 12/30 vs 4/240, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 1.54e-15) and
dysembryoplastic neuroepithelial tumors (DNET) (10/30 vs. 2/240, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 1.67e-14) in <abbr title="Glioma, IDH-mutant, 1p19q codeletion or dysembryoplastic neuroepithelial tumors">T046 GLI LG IDHmut CODEL/DNET</abbr>  (n = 30)
from the rest of the gliomas in <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion mix A">T047 GLI IHDmut CODEL A</abbr>  (n = 240). There is a significant age disparity between the two clusters
(14.11 vs. 41 y.o., <abbr title="Mann-Whitney U test p-value">MWU p-val</abbr> = 9.61e-11) as the former class is made up entirely of pediatric samples.
<abbr title="Glioblastoma, IDH-mutant 1p19q codeletion mix A">T047</abbr> further split by age and histotype. <abbr title="Glioma, IDH-mutant, multicellular network">T048  GLI IHDmut MULTICELL NET</abbr> (n = 67) has significantly younger patients than <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion mix B">T049 GLI IHDmut CODEL B</abbr> (n = 173)
(median 35.00 vs 44.00 y.o. <abbr title="Mann-Whitney U test p-value">MWU p-val</abbr> = 2.26e-03) due to its larger pediatric component (38.80% vs. 11.56%, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 3.70e-06). There is no
difference in overall survial (<abbr title="Kaplan-Meier log rank test p-value">lrt p-val</abbr> = 6.23e-02 at 5546 days). While <abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr> contains more astrocytomas (13/48 vs 4/150, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 7.06e-07),
<abbr title="Glioblastoma, IDH-mutant 1p19q codeletion mix B">T049</abbr> has a considerably higher oligodendroglioma population (20/48 vs 114/150, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 2.14e-05). According to TCGA data, <abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr> is enriched for
<em>EGFR</em> (3/33 vs. 0/87, <abbr title="χ2 test p-value">χ2 p-val</abbr>  = 2.83e-02), <em>ATRX</em> (12/33 vs. 6/87, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 1.77e-04) and <em>TP53</em>-mutant tumors (13/33 vs. 7/87, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 1.23e-04),
while <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion mix B">T049</abbr> contains more <em>CIC</em> (<abbr title="χ2 test p-value">χ2 p-val</abbr> = 1/33 vs. 48/87, 6.33e-07), <em>FUBP1</em> (1/33 vs. 22/87, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 1.22e-02) and <em>NOTCH1</em> (0/33 vs. 19/87, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 8.14e-03)
mutants (Fig. <a class="reference internal" href="#fig-cns7"><span class="std std-ref">CNS7</span></a>). Most tumors in <abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr> are neural (46/49 vs. 38/144, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 7.39e-16), while the majority of those in <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion mix B">T049</abbr> are proneural
(2/49 vs. 105/144, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 2.26e-16). Most importantly, and quite unexpectedly, <abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr> is mostly composed of IDH1 wild-type (16/53) and non-codeleted
samples (45/53). It is not clear why this class is found within the IDH1-codeleted branch.</p>
<div class="figure align-default" id="fig-cns7">
<a class="reference internal image-reference" href="../_images/cns7.png"><img alt="Fig. CNS7" src="../_images/cns7.png" style="width: 800px;" /></a>
<p class="caption"><span class="caption-text">CNS7: Top and center, per sample Normalized Enrichment Score (NES) distributions of gene sets characterizing the gliomas subtype with multicellular network overexpression.
Bottom, per sample expression distributions of <em>NOTCH1</em> and <em>GAP43</em> genes in gliomas subtypes with or without multicellular network overexpression.</span><a class="headerlink" href="#fig-cns7" title="Permalink to this image">¶</a></p>
</div>
<p>Gene set enrichment analysis reveals that every locus available for chr1p, with the expectation of chr1p11, (<abbr title="Mann-Whitney U test p-value">MWU p-val</abbr> ≤ 8.35e-04) and chr19q
(<abbr title="Mann-Whitney U test p-value">MWU p-val</abbr> ≤ 5.18e-23) are significantly downregulated in <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion mix B">T049</abbr> compared to <abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr>, confirming more severe population-wide loss of these loci in
<abbr title="Glioblastoma, IDH-mutant 1p19q codeletion mix B">T049</abbr> vs. <abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr> and supporting that, true to their annotation, the majority of samples in <abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr> have normal expression of these loci, in spite of
their transcriptional similarities with the codeleted branch. The overall expression profile of both IDH wild type and non-codeleted tumors within
<abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr> have a high correlation with true chr1p/19q co-deleted IDHmut gliomas within <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion">T044</abbr> (R ≥ 0.802, Pearson correlation p-val &lt; 2.20e-16).
Further examination of gene sets upregulated in <abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr> compared to its sibling class <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion mix B">T049</abbr>, its uncle class <abbr title="Glioblastoma, IDH-mutant without 1p19q codeletion">T045</abbr> GLI IDHmut noCODEL, and its cousin
class <abbr title="Glioma, IDH wild-type">T030 GLI IDHwt</abbr> revealed significant upregulation of genesets related to neuron-neuron synaptic transmission (<abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 2.35e-89,
<abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.13, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr>-val &lt; 1.00e-04), synaptic plasticity (<abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 5.23e-85, <abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.30, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr>-val &lt; 1.00e-04),
neurite formation (<abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 2.45e-51, <abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.15, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 1.00e-04) <a class="reference internal" href="T001_nebla.html#jassal2020" id="id36"><span>[Jassal2020]</span></a>, and microtubule polymerization
(<abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 1.59e-63, <abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.10, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr>-val &lt; 1.00e-04) (Fig. <a class="reference internal" href="#fig-cns7"><span class="std std-ref">CNS7</span></a>). We also observe upregulation of glutaminergic signalling (<abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 1.06e-102,
<abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.31, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value)`&lt; 1.00e-04), particularly of AMPA cationic channel activity (:abbr:`KW adj. p-val (Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> = 9.08e-59, <abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.43, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 1.00e-04),
including AMPA-dependent synaptic plasticity (<abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 7.36-84, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 1.00e-04), and of extracellular calcium export (<abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 7.61e-93,
<abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 1.00e-04, <abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.14) <a class="reference internal" href="#venkataramani2019" id="id37"><span>[Venkataramani2019]</span></a>, <a class="reference internal" href="#venkatesh2019" id="id38"><span>[Venkatesh2019]</span></a>, <a class="reference internal" href="T001_nebla.html#jassal2020" id="id39"><span>[Jassal2020]</span></a> (Fig. <a class="reference internal" href="#fig-cns7"><span class="std std-ref">CNS7</span></a>).
We also observe increases in gap junction formation (<abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 5.81e-35, <abbr title="median Normalized Enrichment Score">medNES</abbr> = 2.39, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 1.00e-04) and connexin binding
(<abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 3.61e-28, <abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.26, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 1.00e-04) (Fig. <a class="reference internal" href="#fig-cns7"><span class="std std-ref">CNS7</span></a>).
Taken together, these results suggest <abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr> to be composed of gliomas of a recently described multicellular network phenotype, a pro-invasive and
radioresistant resistant mode of glioma growth <a class="reference internal" href="#osswald2015" id="id40"><span>[Osswald2015]</span></a>. Gene expression analysis reveals significant upregulation of <em>GAP34</em> in <abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr>
vs. other IDH1-mutant tumor groups (<abbr title="Glioblastoma, IDH-mutant without 1p19q codeletion">T045</abbr> and <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion mix B">T049</abbr>) (<abbr title="median Log-fold Change">medLogFC</abbr> = 1.33, <abbr title="False Discovery Rate">FDR</abbr> ≤ 1.80e-13), the principal gap-junction protein mediating this
phenotype <a class="reference internal" href="#osswald2015" id="id41"><span>[Osswald2015]</span></a>, as well as <em>NOTCH1</em> underexpression (<abbr title="median Log-fold Change)`= -1.16, :abbr:`FDR (False Discovery Rate">medLogFC</abbr> ≤ 1.950e-06) and downregulation of <em>NOTCH1</em> signalling
(<abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 1.65e-45, <abbr title="median Normalized Enrichment Score">medNES</abbr> = 0.92, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr>-val &lt; 0.05) <a class="reference internal" href="T001_nebla.html#jassal2020" id="id42"><span>[Jassal2020]</span></a> over all other glioma types, the crucial determinant of this phenotype <a class="reference internal" href="#jung2021" id="id43"><span>[Jung2021]</span></a> (Fig. <a class="reference internal" href="#fig-cns7"><span class="std std-ref">CNS7</span></a>).
This is despite the lack of <em>NOTCH1</em>-mutant samples in <abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr>; <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion mix B">T049</abbr> contains the majority NOTCH1 mutants of the glioma cohort (vs. <abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr>, <abbr title="Glioblastoma, IDH-mutant without 1p19q codeletion">T045</abbr>, and T030, 0/33 vs. 19/87 vs. 4/113 vs. 0/52, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 2.29e-07)
and exhibits the highest <em>NOTCH1</em> expression (<abbr title="median Log-fold Change)`= 0.83, :abbr:`FDR (False Discovery Rate">medLogFC</abbr> ≤ 2.961e-02), so we speculate these NOTCH1 mutations to be gain-of-function. However, despite this phenotype displaying radioresistance, samples in <abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr> show no significant differences in overall survival compared to other IDH-mutated glioma groups (<abbr title="Glioblastoma, IDH-mutant without 1p19q codeletion">T045</abbr> and <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion mix B">T049</abbr>) at 6423 days.
We speculate this novel phenotype may have good transcriptional affinity with chr1p/19q codeletion, in spite of the lack of apparent lesions
in the region.
Though this phenotype is mostly associated with astrocytomas <a class="reference internal" href="#osswald2015" id="id44"><span>[Osswald2015]</span></a>,  <abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr> is a mixed cluster – containing large amounts of both
astrocytomas and oligodendrogliomas. <abbr title="Glioma, IDH-mutant, multicellular network">T048</abbr> then splits in two classes (Fig. <a class="reference internal" href="#fig-cns1"><span class="std std-ref">CNS1</span></a>), with different histological populations;
<abbr title="Glioma, IDH-mutant, multicellular network, oligodendroglioma">T050 GLI IDHmut MULTICELL NET OLIGOD</abbr>  (n = 31) contains more oligodendrogliomas (15/22 vs. 5/26, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 1.73-3) than <abbr title="Glioma, IDH-mutant, multicellular network, astrocytoma">T051 GLI IDHmut MULTICELL NET ASTROC</abbr>  (n = 36),
which instead is populated by astrocytomas (0/22 vs. 13/26, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 3.74e-04) <a class="reference internal" href="#davare2018" id="id45"><span>[Davare2018]</span></a>, <a class="reference internal" href="#zhang2017" id="id46"><span>[Zhang2017]</span></a>.
<abbr title="Glioma, IDH-mutant, multicellular network, oligodendroglioma">T050</abbr> also inherits the
majority of chr1p/19q codelted samples (7/16 vs. 1/29, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 1.91e-02).
Similarly, <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion mix B">T049</abbr> splits by histological composition (Fig. <a class="reference internal" href="#fig-cns1"><span class="std std-ref">CNS1</span></a>) with <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion NOTCH1 mutant">T052 GLI IHDmut CODEL NOTCH1</abbr> (n = 89) being enriched (68/81 vs. 46/69,
<abbr title="χ2 test p-value">χ2 p-val</abbr> = 2.27e-02) for oligodendrogliomas and <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion oligoastrocytomat">T053 GLI IHDmut CODEL OLIGOAST</abbr> (n = 84) for oligoastrocytomas (11/81 vs. 21/69, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 2.08e-02).
<abbr title="Glioblastoma, IDH-mutant 1p19q codeletion NOTCH1 mutant">T052</abbr> also has a significantly higher population of <em>NOTCH1</em> mutant samples (17/56 vs. 2/39, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 2.07e-02). <abbr title="Glioma, IDH-mutant, multicellular network, astrocytoma">T051</abbr> contains a larger share of
neural gliomas (30/78 vs. 8/66, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 7.154e-04), while <abbr title="Glioblastoma, IDH-mutant 1p19q codeletion NOTCH1 mutant">T052</abbr> contains more proneural gliomas (48/78 vs. 57/66, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 1.623-03).
Finally, following along the non-codeleted branch defined by <abbr title="Glioblastoma, IDH-mutant without 1p19q codeletion">T045</abbr>, we observe four children classes, characterized by significant differences in
the sex ratios (Fig. <a class="reference internal" href="#fig-cns1"><span class="std std-ref">CNS1</span></a>): <abbr title="Glioblastoma, IDH-mutant without 1p19q codeletion oligodendrocytes">T054 GLI IDHmut noCODEL OLIGOC</abbr>  (n = 99) is composed by 63.64% of males,
<abbr title="Glioma, IDH-mutant, no codeletion, neural development">T055 GLI IDHmut noCODEL NEUR DIFFhigh</abbr>  (n = 30)
is entirely female (<abbr title="χ2 test p-value">χ2 p-val</abbr> = 2.65e-17), <abbr title="Glioma, IDH-mutant, no codeletion, mesenchymal">T056 GLI IDHmut noCODEL MES</abbr>  (n = 24) is 75.00% male and <abbr title="Glioma, IDH-mutant, no codeletion, H3 demethylation">T057 GLI IDHmut noCODEL H3demet</abbr> (n = 47) is almost
exclusively male (97.87% <abbr title="χ2 test p-value">χ2 p-val</abbr> = 2.65e-17). Although sex differences have previously been reported to be associated with differences in survival
in glioma and <abbr title="Glioblastoma">GBM</abbr> <a class="reference internal" href="#yang2019" id="id47"><span>[Yang2019]</span></a>, patients in these clusters have no significant differences in overall survival (<abbr title="Kaplan-Meier log rank test p-value">lrt p-val</abbr> = 1.44e-01 at 4752 days).</p>
<p>Differential expression and gene sets analyses on these classes revelated that <abbr title="Glioblastoma, IDH-mutant without 1p19q codeletion oligodendrocytes">T054</abbr> is enriched for oligodendrocyte development
(<abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.03, adj p-val = 4.85e-15) and myelination (<abbr title="median Normalized Enrichment Score">medNES</abbr> = 1.23, <abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 3.97e-09, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 0.01) <a class="reference internal" href="T002_T003_mesoderm.html#tgoc2019" id="id48"><span>[TGOC2019]</span></a>, <a class="reference internal" href="T005_leuk.html#ashburner2000" id="id49"><span>[Ashburner2000]</span></a>,
and overexpresses <em>MBP</em> and <em>MOBP</em> (<abbr title="False Discovery Rate">FDR</abbr> ≤ 2.461e-04), predictors of improved survival <a class="reference internal" href="#wang2019" id="id50"><span>[Wang2019]</span></a>, <a class="reference internal" href="#kong2013" id="id51"><span>[Kong2013]</span></a>. <abbr title="Glioma, IDH-mutant, no codeletion, neural development">T055</abbr> is enriched for gene sets related
to neuronal development (<abbr title="median Normalized Enrichment Score">medNES</abbr> ≥ 1.01, <abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 9.36e-17). <abbr title="Glioma, IDH-mutant, no codeletion, mesenchymal">T056</abbr> is enriched for genesets involving <em>MYC</em> signalling (<abbr title="median Normalized Enrichment Score">medNES</abbr> ≥ 1.04,
<abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> ≤ 6.76e-03, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 0.05), the G2M checkpoint,(<abbr title="median Normalized Enrichment Score">medNES</abbr> ≥ 1.16,
<abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 1.46e-03, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 0.01), and the
immune response (<abbr title="median Normalized Enrichment Score">medNES</abbr> ≥ 1.510, <abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> ≤ 1.05e-05, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 0.01) <a class="reference internal" href="T002_T003_mesoderm.html#liberzon2015" id="id52"><span>[Liberzon2015]</span></a>; it also exhibits the highest immune infiltration
score out of its siblings (median = 990 vs. 648 vs. 2185 vs. 977, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> ≤ 4.81e-05). It overexpresses MMP9 (<abbr title="median Log-fold Change">medLogFC</abbr> = 1.41, <abbr title="False Discovery Rate">FDR</abbr> ≤ 3.515e-02),
<em>CHI3L1</em> (<abbr title="median Log-fold Change">medLogFC</abbr> = 1.79, <abbr title="False Discovery Rate">FDR</abbr> 8.332e-03), S100A4 (<abbr title="median Log-fold Change)`= 2.12, :abbr:`FDR (False Discovery Rate">medLogFC</abbr> ≤ 4.281e-09), EN1 (<abbr title="median Log-fold Change)`= 5.01, :abbr:`FDR (False Discovery Rate">medLogFC</abbr> ≤ 8.643e-14), and <em>ANXA1</em>
(<abbr title="median Log-fold Change)`= 2.56, :abbr:`FDR (False Discovery Rate">medLogFC</abbr> ≤ 7.266e-14), markers of poor prognosis, and <em>IGF2BP3</em> (<abbr title="median Log-fold Change)`= 4.06, :abbr:`FDR (False Discovery Rate">medLogFC</abbr> ≤ 8.147e-13), a glioblastoma-specific proliferative and
invasive marker. <abbr title="Glioma, IDH-mutant, no codeletion, mesenchymal">T056</abbr> is also the only cluster to contain a significant population of mesenchymal samples (0/58 vs. 0/21 vs. 5/11 vs. 2/ 35, <abbr title="χ2 test p-value">χ2 p-val</abbr> = 3.211e-08)
and is enriched for epithelial mesenchymal transition genesets (<abbr title="median Normalized Enrichment Score">medNES</abbr> ≥ 1.15, <abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> = 4.46e-07, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 0.05) <a class="reference internal" href="T002_T003_mesoderm.html#liberzon2015" id="id53"><span>[Liberzon2015]</span></a>.
<abbr title="Glioma, IDH-mutant, no codeletion, H3 demethylation">T057</abbr> is enriched for genesets involving <em>H3K4</em> demethylation (<abbr title="median Normalized Enrichment Score">medNES</abbr> ≥ 1.07, <abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> ≤ 6.91e-11, <abbr title="Dunn’s test of multiple comparisons Benjamin-Hochberg adjusted p-value">Dunn adj. p-val</abbr> &lt; 0.05) and <em>H3K27</em> demethylation
(<abbr title="median Normalized Enrichment Score">medNES</abbr> ≥ 1.08, <abbr title="Kruskal–Wallis one-way analysis of variance test Benjamin-Hochberg adjusted p-value">KW adj. p-val</abbr> ≤ 6.00e-03) <a class="reference internal" href="T002_T003_mesoderm.html#tgoc2019" id="id54"><span>[TGOC2019]</span></a>, <a class="reference internal" href="T005_leuk.html#ashburner2000" id="id55"><span>[Ashburner2000]</span></a>. It also overexpresses <em>LDHC</em> (median LogFC= 3.44,
<abbr title="False Discovery Rate">FDR</abbr> ≤ 7.16e-06), which was found to be elevated in mesenchymal glioma stem cells and negatively correlates with survival <a class="reference internal" href="#mao2013" id="id56"><span>[Mao2013]</span></a>, <a class="reference internal" href="#beckner2016" id="id57"><span>[Beckner2016]</span></a>.</p>
</div>
<div class="section" id="bibliography">
<h2>Bibliography<a class="headerlink" href="#bibliography" title="Permalink to this headline">¶</a></h2>
<dl class="citation">
<dt class="label" id="ashburner2000"><span class="brackets">Ashburner2000</span><span class="fn-backref">(<a href="#id35">1</a>,<a href="#id49">2</a>,<a href="#id55">3</a>)</span></dt>
<dd><p>Ashburner, M., Ball, C.A., Blake, J.A.,2000. Gene Ontology: tool for the unification of biology. Nature Genetics 25(1), pp. 25–29.</p>
</dd>
<dt class="label" id="beck2011"><span class="brackets"><a class="fn-backref" href="#id19">Beck2011</a></span></dt>
<dd><p>Beck, S., Jin, X., Sohn, Y.-W.,2011. Telomerase activity-independent function of TERT allows glioma cells to attain cancer stem cell characteristics by inducing EGFR expression. Molecules and Cells 31(1), pp. 9–15.</p>
</dd>
<dt class="label" id="beckner2016"><span class="brackets"><a class="fn-backref" href="#id57">Beckner2016</a></span></dt>
<dd><p>Beckner, M.E., Pollack, I.F., Nordberg, M.L. and Hamilton, R.L. 2016. Glioblastomas with copy number gains in EGFR and RNF139 show increased expressions of carbonic anhydrase genes transformed by ENO1. BBA clinical 5, pp. 1–15.</p>
</dd>
<dt class="label" id="brennan2013"><span class="brackets"><a class="fn-backref" href="#id22">Brennan2013</a></span></dt>
<dd><p>Brennan, C.W., Verhaak, R.G.W., McKenna, A.,2013. The somatic genomic landscape of glioblastoma. Cell 155(2), pp. 462–477.</p>
</dd>
<dt class="label" id="ceccarelli2016"><span class="brackets">Ceccarelli2016</span><span class="fn-backref">(<a href="#id20">1</a>,<a href="#id21">2</a>,<a href="#id27">3</a>,<a href="#id28">4</a>)</span></dt>
<dd><p>Ceccarelli, M., Barthel, F.P., Malta, T.M.,2016. Molecular profiling reveals biologically discrete subsets and pathways of progression in diffuse glioma. Cell 164(3), pp. 550–563.</p>
</dd>
<dt class="label" id="chen2020"><span class="brackets"><a class="fn-backref" href="#id32">Chen2020</a></span></dt>
<dd><p>Chen, K.-Y., Bush, K., Klein, R.H.,2020. Reciprocal H3.3 gene editing identifies K27M and G34R mechanisms in pediatric glioma including NOTCH signaling. Communications Biology 3(1), p. 363.</p>
</dd>
<dt class="label" id="cimino2018"><span class="brackets">Cimino2018</span><span class="fn-backref">(<a href="#id7">1</a>,<a href="#id17">2</a>)</span></dt>
<dd><p>Cimino, P.J., Kim, Y., Wu, H.-J.,2018. Increased HOXA5 expression provides a selective advantage for gain of whole chromosome 7 in IDH wild-type glioblastoma. Genes &amp; Development 32(7–8), pp. 512–523.</p>
</dd>
<dt class="label" id="davare2018"><span class="brackets"><a class="fn-backref" href="#id45">Davare2018</a></span></dt>
<dd><p>Davare, M.A., Henderson, J.J., Agarwal, A.,2018. Rare but recurrent ROS1 fusions resulting from chromosome 6q22 microdeletions are targetable oncogenes in glioma. Clinical Cancer Research 24(24), pp. 6471–6482.</p>
</dd>
<dt class="label" id="fadhlullah2019"><span class="brackets"><a class="fn-backref" href="#id33">Fadhlullah2019</a></span></dt>
<dd><p>Fadhlullah, S.F.B., Halim, N.B.A., Yeo, J.Y.T.,2019. Pathogenic mutations in neurofibromin identifies a leucine-rich domain regulating glioma cell invasiveness. Oncogene 38(27), pp. 5367–5380.</p>
</dd>
<dt class="label" id="gearhart2006"><span class="brackets"><a class="fn-backref" href="#id9">Gearhart2006</a></span></dt>
<dd><p>Gearhart, M.D., Corcoran, C.M., Wamstad, J.A. and Bardwell, V.J. 2006. Polycomb group and SCF ubiquitin ligases are found in a novel BCOR complex that is recruited to BCL6 targets. Molecular and Cellular Biology 26(18), pp. 6880–6889.</p>
</dd>
<dt class="label" id="hanzelmann2013"><span class="brackets"><a class="fn-backref" href="#id2">Hänzelmann2013</a></span></dt>
<dd><p>Hänzelmann, S., Castelo, R. and Guinney, J. 2013. GSVA: gene set variation analysis for microarray and RNA-seq data. BMC Bioinformatics 14, p. 7.</p>
</dd>
<dt class="label" id="hartmann2010"><span class="brackets"><a class="fn-backref" href="#id4">Hartmann2010</a></span></dt>
<dd><p>Hartmann, C., Hentschel, B., Wick, W.,2010. Patients with IDH1 wild type anaplastic astrocytomas exhibit worse prognosis than IDH1-mutated glioblastomas, and IDH1 mutation status accounts for the unfavorable prognostic effect of higher age: implications for classification of gliomas. Acta Neuropathologica 120(6), pp. 707–718.</p>
</dd>
<dt class="label" id="hernandez2010"><span class="brackets"><a class="fn-backref" href="#id8">Hernández2010</a></span></dt>
<dd><p>Hernández, M., Martín, R., García-Cubillas, M.D., Maeso-Hernández, P. and Nieto, M.L. 2010. Secreted PLA2 induces proliferation in astrocytoma through the EGF receptor: another inflammation-cancer link. Neuro-oncology 12(10), pp. 1014–1023.</p>
</dd>
<dt class="label" id="jassal2020"><span class="brackets">Jassal2020</span><span class="fn-backref">(<a href="#id36">1</a>,<a href="#id39">2</a>,<a href="#id42">3</a>)</span></dt>
<dd><p>Jassal, B., Matthews, L., Viteri, G.,2020. The Reactome Pathway Knowledgebase. Nucleic Acids Research 48(D1), pp. D498–D503.</p>
</dd>
<dt class="label" id="jung2021"><span class="brackets"><a class="fn-backref" href="#id43">Jung2021</a></span></dt>
<dd><p>Jung, E., Osswald, M., Ratliff, M.,2021. Tumor cell plasticity, heterogeneity, and resistance in crucial microenvironmental niches in glioma. Nature Communications 12(1), p. 1014.</p>
</dd>
<dt class="label" id="kanehisa2000"><span class="brackets">Kanehisa2000</span><span class="fn-backref">(<a href="#id3">1</a>,<a href="#id13">2</a>)</span></dt>
<dd><p>Kanehisa, M. and Goto, S. 2000. KEGG: Kyoto encyclopedia of genes and genomes. Nucleic Acids Research 28(1), pp. 27–30.</p>
</dd>
<dt class="label" id="kao2018"><span class="brackets"><a class="fn-backref" href="#id14">Kao2018</a></span></dt>
<dd><p>Kao, Y.-C., Owosho, A.A., Sung, Y.-S.,2018. BCOR-CCNB3 Fusion Positive Sarcomas: A Clinicopathologic and Molecular Analysis of 36 Cases With Comparison to Morphologic Spectrum and Clinical Behavior of Other Round Cell Sarcomas. The American Journal of Surgical Pathology 42(5), pp. 604–615.</p>
</dd>
<dt class="label" id="kao2020"><span class="brackets"><a class="fn-backref" href="#id15">Kao2020</a></span></dt>
<dd><p>Kao, Y.-C., Sung, Y.-S., Argani, P.,2020. NTRK3 overexpression in undifferentiated sarcomas with YWHAE and BCOR genetic alterations. Modern Pathology 33(7), pp. 1341–1349.</p>
</dd>
<dt class="label" id="kong2013"><span class="brackets"><a class="fn-backref" href="#id51">Kong2013</a></span></dt>
<dd><p>Kong, J., Cooper, L.A.D., Wang, F.,2013. Machine-based morphologic analysis of glioblastoma using whole-slide pathology images uncovers clinically relevant molecular correlates. Plos One 8(11), p. e81049.</p>
</dd>
<dt class="label" id="larson2019"><span class="brackets"><a class="fn-backref" href="#id29">Larson2019</a></span></dt>
<dd><p>Larson, J.D., Kasper, L.H., Paugh, B.S.,2019. Histone H3.3 K27M accelerates spontaneous brainstem glioma and drives restricted changes in bivalent gene expression. Cancer Cell 35(1), p. 140–155.e7.</p>
</dd>
<dt class="label" id="liberzon2015"><span class="brackets">Liberzon2015</span><span class="fn-backref">(<a href="#id11">1</a>,<a href="#id52">2</a>,<a href="#id53">3</a>)</span></dt>
<dd><p>Liberzon, A., Birger, C., Thorvaldsdóttir, H., Ghandi, M., Mesirov, J.P. and Tamayo, P. 2015. The Molecular Signatures Database (MSigDB) hallmark gene set collection. Cell Systems 1(6), pp. 417–425.</p>
</dd>
<dt class="label" id="mao2013"><span class="brackets"><a class="fn-backref" href="#id56">Mao2013</a></span></dt>
<dd><p>Mao, P., Joshi, K., Li, J.,2013. Mesenchymal glioma stem cells are maintained by activated glycolytic metabolism involving aldehyde dehydrogenase 1A3. Proceedings of the National Academy of Sciences of the United States of America 110(21), pp. 8644–8649.</p>
</dd>
<dt class="label" id="nishimura2001"><span class="brackets"><a class="fn-backref" href="#id12">Nishimura2001</a></span></dt>
<dd><p>Nishimura, D. 2001. BioCarta. Biotech Software &amp; Internet Report 2(3), pp. 117–120.</p>
</dd>
<dt class="label" id="northcott2012"><span class="brackets"><a class="fn-backref" href="#id1">Northcott2012</a></span></dt>
<dd><p>Northcott, P.A., Dubuc, A.M., Pfister, S. and Taylor, M.D. 2012. Molecular subgroups of medulloblastoma. Expert Review of Neurotherapeutics 12(7), pp. 871–884.</p>
</dd>
<dt class="label" id="osswald2015"><span class="brackets">Osswald2015</span><span class="fn-backref">(<a href="#id40">1</a>,<a href="#id41">2</a>,<a href="#id44">3</a>)</span></dt>
<dd><p>Osswald, M., Jung, E., Sahm, F.,2015. Brain tumor cells interconnect to a functional and resistant network. Nature 528(7580), pp. 93–98.</p>
</dd>
<dt class="label" id="park2016"><span class="brackets"><a class="fn-backref" href="#id5">Park2016</a></span></dt>
<dd><p>Park, S.Y., Piao, Y., Jeong, K.J., Dong, J. and de Groot, J.F. 2016. Periostin (POSTN) regulates tumor resistance to antiangiogenic therapy in glioma models. Molecular Cancer Therapeutics 15(9), pp. 2187–2197.</p>
</dd>
<dt class="label" id="paugh2011"><span class="brackets"><a class="fn-backref" href="#id31">Paugh2011</a></span></dt>
<dd><p>Paugh, B.S., Broniscer, A., Qu, C.,2011. Genome-wide analyses identify recurrent amplifications of receptor tyrosine kinases and cell-cycle regulatory genes in diffuse intrinsic pontine glioma. Journal of Clinical Oncology 29(30), pp. 3999–4006.</p>
</dd>
<dt class="label" id="roversi2006"><span class="brackets"><a class="fn-backref" href="#id26">Roversi2006</a></span></dt>
<dd><p>Roversi, G., Pfundt, R., Moroni, R.F.,2006. Identification of novel genomic markers related to progression to glioblastoma through genomic profiling of 25 primary glioma cell lines. Oncogene 25(10), pp. 1571–1583.</p>
</dd>
<dt class="label" id="schaefer2009"><span class="brackets"><a class="fn-backref" href="#id30">Schaefer2009</a></span></dt>
<dd><p>Schaefer, C.F., Anthony, K., Krupa, S.,2009. PID: the pathway interaction database. Nucleic Acids Research 37(Database issue), pp. D674-9.</p>
</dd>
<dt class="label" id="steponaitis2016"><span class="brackets"><a class="fn-backref" href="#id6">Steponaitis2016</a></span></dt>
<dd><p>Steponaitis, G., Skiriutė, D., Kazlauskas, A.,2016. High CHI3L1 expression is associated with glioma patient survival. Diagnostic Pathology 11, p. 42.</p>
</dd>
<dt class="label" id="sturm2016"><span class="brackets"><a class="fn-backref" href="#id10">Sturm2016</a></span></dt>
<dd><p>Sturm, D., Orr, B.A., Toprak, U.H.,2016. New Brain Tumor Entities Emerge from Molecular Classification of CNS-PNETs. Cell 164(5), pp. 1060–1072.</p>
</dd>
<dt class="label" id="tabuse2011"><span class="brackets"><a class="fn-backref" href="#id16">Tabuse2011</a></span></dt>
<dd><p>Tabuse, M., Ohta, S., Ohashi, Y.,2011. Functional analysis of HOXD9 in human gliomas and glioma cancer stem cells. Molecular Cancer 10, p. 60.</p>
</dd>
<dt class="label" id="tachon2019"><span class="brackets"><a class="fn-backref" href="#id23">Tachon2019</a></span></dt>
<dd><p>Tachon, G., Masliantsev, K., Rivet, P.,2019. Prognostic significance of MEOX2 in gliomas. Modern Pathology 32(6), pp. 774–786.</p>
</dd>
<dt class="label" id="tgoc2019"><span class="brackets">TGOC2019</span><span class="fn-backref">(<a href="#id34">1</a>,<a href="#id48">2</a>,<a href="#id54">3</a>)</span></dt>
<dd><p>The Gene Ontology Consortium 2019. The Gene Ontology Resource: 20 years and still GOing strong. Nucleic Acids Research 47(D1), pp. D330–D338.</p>
</dd>
<dt class="label" id="thorsson2018"><span class="brackets"><a class="fn-backref" href="#id25">Thorsson2018</a></span></dt>
<dd><p>Thorsson, V., Gibbs, D.L., Brown, S.D.,2018. The immune landscape of cancer. Immunity 48(4), p. 812–830.e14.</p>
</dd>
<dt class="label" id="venkataramani2019"><span class="brackets"><a class="fn-backref" href="#id37">Venkataramani2019</a></span></dt>
<dd><p>Venkataramani, V., Tanev, D.I., Strahle, C.,2019. Glutamatergic synaptic input to glioma cells drives brain tumor progression. Nature 573(7775), pp. 532–538.</p>
</dd>
<dt class="label" id="venkatesh2019"><span class="brackets"><a class="fn-backref" href="#id38">Venkatesh2019</a></span></dt>
<dd><p>Venkatesh, H.S., Morishita, W., Geraghty, A.C.,2019. Electrical and synaptic integration of glioma into neural circuits. Nature 573(7775), pp. 539–545.</p>
</dd>
<dt class="label" id="wang2019"><span class="brackets"><a class="fn-backref" href="#id50">Wang2019</a></span></dt>
<dd><p>Wang, S., Liu, F., Wang, Y.,2019. Integrated analysis of 34 microarray datasets reveals CBX3 as a diagnostic and prognostic biomarker in glioblastoma. Journal of Translational Medicine 17(1), p. 179.</p>
</dd>
<dt class="label" id="xu2017"><span class="brackets">Xu2017</span><span class="fn-backref">(<a href="#id18">1</a>,<a href="#id24">2</a>)</span></dt>
<dd><p>Xu, Yonggang, Wang, J., Xu, Yanbin, Xiao, H., Li, J. and Wang, Z. 2017. Screening critical genes associated with malignant glioma using bioinformatics analysis. Molecular medicine reports 16(5), pp. 6580–6589.</p>
</dd>
<dt class="label" id="yang2019"><span class="brackets"><a class="fn-backref" href="#id47">Yang2019</a></span></dt>
<dd><p>Yang, W., Warrington, N.M., Taylor, S.J.,2019. Sex differences in GBM revealed by analysis of patient imaging, transcriptome, and survival data. Science Translational Medicine 11(473).</p>
</dd>
<dt class="label" id="zhang2017"><span class="brackets"><a class="fn-backref" href="#id46">Zhang2017</a></span></dt>
<dd><p>Zhang, X., Lv, Q.-L., Huang, Y.-T., Zhang, L.-H. and Zhou, H.-H. 2017. Akt/FoxM1 signaling pathway-mediated upregulation of MYBL2 promotes p</p>
</dd>
</dl>
</div>
</div>


          </div>
          <div class="page-nav">
            <div class="inner"><ul class="page-nav">
  <li class="prev">
    <a href="N000_normal.html"
       title="previous chapter">← N000 Healthy Normal Tissue</a>
  </li>
  <li class="next">
    <a href="T001_nebla.html"
       title="next chapter">T001 Neuroblastoma →</a>
  </li>
</ul><div class="footer" role="contentinfo">
      &#169; Copyright 2022, Federico Comitani, Joshua O. Nash.
    <br>
    Created using <a href="http://sphinx-doc.org/">Sphinx</a> 4.3.2 with <a href="https://github.com/schettino72/sphinx_press_theme">Press Theme</a> 0.8.0.
</div>
            </div>
          </div>
      </page>
    </div></div>
    
    
  </body>
</html>